serving/pom.xml

```diff
@@ -342,6 +348,12 @@
       <version>1.15.2</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
```

Comment:
We should avoid using cassandra container as it pulls in an older version of cassandra client as dependency.

Reply:
Forgot to remove, it's actually not in use.